zenith/src/main.rs

```diff
@@ -347,7 +339,7 @@ fn parse_point_in_time(s: &str) -> Result<local_env::PointInTime> {
 
         let mut result = parse_point_in_time(&pointstr)?;
 
-        result.lsn = lsn.unwrap_or(0);
+        result.lsn = lsn.unwrap_or(Lsn(0));
```

Comment:
I'm curious why the result might be `Lsn(0)`?  Should this be `Option<Lsn>`?

Reply:
Yeah, that probably would be more idiomatic Rust than using 0 as a special value.